src/schemas/player.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Player",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "role": {
+      "enum": [
+        "WR",
+        "TE",
+        "QB",
+        "RB",
+        "K"
+      ]
+    },
+    "scores": {
+      "type": "object",
+      "properties": {
+        "catches": {
+          "type": "number"
+        },
+        "recYds": {
+          "type": "number"
+        },
+        "recTouchdowns": {
+          "type": "number"
+        },
+        "rushTouchdowns": {
+          "type": "number"
+        },
+        "rushYds": {
+          "type": "number"
+        },
+        "passYds": {
+          "type": "number"
+        },
+        "passTouchdowns": {
+          "type": "number"
+        },
+        "fumbles": {
+          "type": "number"
+        },
+        "fumblesLost": {
+          "type": "number"
+        }
```

Comment:
If this player schema is for kickers too, we need to add their FGs, PATs, FG missed

Reply:
Also we may need to consider if we're doing these as season or weekly totals, and if we need to record both

Reply:
I think the schema can represent a single game total - season/multi-week aggregations can be done in the UI.

Reply:
Awesome